src/sentry/runner/commands/devservices.py

```
@@ -455,18 +462,9 @@ def _down(container: docker.models.containers.Container) -> None:
             for container in containers:
                 futures.append(executor.submit(_down, container))
             for future in as_completed(futures):
-                # If there was an exception, reraising it here to the main thread
-                # will not terminate the whole python process. We'd like to report
-                # on this exception and stop as fast as possible, so terminate
-                # ourselves. I believe (without verification) that the OS is now
-                # free to cleanup these threads, but not sure if they'll remain running
-                # in the background. What matters most is that we regain control
-                # of the terminal.
                 e = future.exception()
                 if e:
-                    click.echo(e)
-                    me = os.getpid()
-                    os.kill(me, signal.SIGTERM)
+                    click.secho(f"> Failed to stop service: {e}", err=True, fg="red")
```

this should still reraise this exception otherwise it will silently fail